postr/inserter.py

```diff
@@ -47,11 +47,12 @@ def create_bio(
         display_last: str,
         age: int,
         comment: str,
+        person_id: int,
```

Comment:
Probably should've been done last week, but can you add some kind of documentation (docstrings) for what this file is as a whole, and some methods? 'inserter.py' isn't that clear. Is it only for twitter?  What's a 'Job' and why are we storing Bios?

Reply:
Yes, it should have been done last week. Inserter inserts rows into the database for scheduling.

Job is a media operation task to do, it takes in various arguments (i.e. intervals, start time).

We are storing bios, so that we can mass update them if needed. For example, if I wanted to change my twitter handle from @adambeck to @beck across all my bios, the database will only return the bios belonging to me that contain @beck. This way, we don't have to parse every website to see if we need to change our bio on that site or not.

Reply:
You rang?